### ISSUE TITLE
Documents delimiter for CoNLL 2003 NER

### ISSUE THREAD

**Issue by abduhbm** [heartexlabs/label-studio-converter#10]
Exported annotated data in CoNLL 2003 NER format cannot be imported in SpaCy. 
SpaCy expects documents to be separated using `-DOCSTART- -X- O O` line and sentences with whitespaces as per its documentation for converting CoNLL-2003 NER format to json. 
https://spacy.io/api/cli#convert

Should this be handled in the converter? If yes, I can push a PR to fix it.

**Comment by normalnyi** [comment]
Yes, it should be implemented here, but I suggest naming it as a different format rather than https://github.com/heartexlabs/label-studio-converter/blob/master/label_studio_converter/converter.py#L34
